includes/admin/views/html-admin-page-status-report.php

```diff
@@ -67,6 +67,11 @@
 		<tr>
 			<td data-export-label="CC Version"><?php esc_html_e( 'Classic Commerce version', 'classic-commerce' ); ?>:</td>
 			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of Classic Commerce installed on your site.', 'classic-commerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
+			<td><?php echo esc_html( $environment['cc_version'] ); ?></td>
+		</tr>
+		<tr>
+			<td data-export-label="WC Version"><?php esc_html_e( 'WooCommerce version', 'classic-commerce' ); ?>:</td>
+			<td class="help"><?php echo wc_help_tip( esc_html__( 'The version of WooCommerce installed on your site.', 'classic-commerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
```

Comment:
Let's use something like - "Equivalent version of Woocommerce installed on your site"

Reply:
OK with me. New commit added.